okio/src/jvmMain/kotlin/okio/AsyncTimeout.kt

```diff
@@ -270,24 +294,22 @@ open class AsyncTimeout : Timeout() {
 
     /** Returns true if the timeout occurred. */
     private fun cancelScheduledTimeout(node: AsyncTimeout): Boolean {
```

Comment:
GitHub’s UI doesn’t make it obvious, but the only change in this function is moving withLock() to the callsites.

Reply:
Looks pretty good with "Hide whitespaces", although I might be biased cause I've already seen this code!